pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -144,10 +144,10 @@ func fixLibvirtInstalled(distro *linux.OsRelease) func() error {
 
 func installLibvirtCommand(distro *linux.OsRelease) string {
 	yumCommand := "yum install -y libvirt libvirt-daemon-kvm qemu-kvm"
-	switch distroID(distro) {
-	case linux.Ubuntu:
+	switch {
+	case distroIsLike(distro, linux.Ubuntu):
 		return "apt-get update && apt-get install -y libvirt-daemon libvirt-daemon-system libvirt-clients"
-	case linux.RHEL, linux.CentOS, linux.Fedora:
+	case distroIsLike(distro, linux.Fedora):
 		return yumCommand
```

Comment:
While you are changing this method, you could probably remove the `yumCommand` variable which doesn't bring much, and is inconsistent with the ubuntu case.

Reply:
I changed the order but for yumCommand, I would have to duplicate in the fedora check and in the default check. Perhaps you have an other idea?